File: app/src/main/java/top/cyixlq/byapptemplate/MyApplication.java
package top.cyixlq.byapptemplate;

import android.app.Application;
import top.cyixlq.network.NetWorkManager;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        NetWorkManager.getInstance().setEnableLog(BuildConfig.DEBUG).init(this);
    }
}

File: app/src/main/java/top/cyixlq/byapptemplate/MyApplication.kt
package top.cyixlq.byapptemplate

import android.app.Application
import top.cyixlq.core.CoreManager
import top.cyixlq.network.ByNetWorkManager


class MyApplication: Application() {

    override fun onCreate() {
        super.onCreate()
        CoreManager.init(this).configNetWork(baseUrl = "http://api.ebuycambodia.com")
            .configCLog(isEnableLog = true, showThreadInfo = false)
        ByNetWorkManager.init()
    }
}